resources/gui/default.theme.txt

```diff
@@ -1,6 +1,8 @@
 [base] {
     font: gui/fonts/BigShouldersDisplay-SemiBold.ttf
-    
+    # Positive offset moves the font down, negative moves it up.
+    offset: 0.0
```

Comment:
I think `font_offset` might be a better name then `offset` as now it isn't clear that it refers to the font